azurelinuxagent/common/protocol/wire.py

```diff
@@ -677,19 +681,21 @@ def _fetch_response(self, uri, headers=None, use_proxy=None):
                 headers=headers,
                 use_proxy=use_proxy)
 
+            host_plugin = self.get_host_plugin()
```

Comment:
Did this to encourage using the get function rather than using the property directly